docs/en/api/wrapper/setProps.md

```diff
@@ -16,5 +16,33 @@ import Foo from './Foo.vue'
 
 const wrapper = mount(Foo)
 wrapper.setProps({ foo: 'bar' })
-expect(wrapper.props().foo).toBe('bar')
+expect(wrapper.vm.foo).to.equal('bar')
+```
+
+You can also pass a `propsData` object, which will initialize the Vue instance with passed values. This is useful if you have a component or instance that has props which are `required`.
```

Comment:
Could you remove this section please"
```
This is useful if you have a component or instance that has props which are `required`.
```

I think the users will understand what it's useful for.